doc/man1/flux-jobs.rst

```diff
@@ -53,6 +53,18 @@ OPTIONS
    Control output coloring. WHEN can be *never*, *always*, or *auto*.
    Defaults to *auto*.
 
+**--stats**
+   Output a summary of global job statistics before the header.
```

Comment:
Should we add some text saying this would be useful with the `watch` command?

Reply:
Eh, I suppose we could, though I was assuming users would figure that out on their own. 

Maybe something like:

```rst
May be useful in conjunction with utilities like ``watch(1)``, e.g.::
 
   $ watch -n2 flux jobs --stats -f running -c 25

will display a summary of global job statistics along with the top 25 running jobs.
```